govc/vm/customize.go

```diff
@@ -69,8 +70,10 @@ func (cmd *customize) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.gateway = nil
 	f.Var(&cmd.netmask, "netmask", "Netmask")
 	cmd.netmask = nil
-	f.Var(&cmd.dnsserver, "dns-server", "DNS server")
+	f.Var(&cmd.dnsserver, "dns-server", "DNS server list")
 	cmd.dnsserver = nil
+	f.Var(&cmd.dnssuffix, "dns-suffix", "DNS suffix list")
+	cmd.dnssuffix = nil
```

Comment:
Is that needed?

Reply:
Not normally, but @gressd has some code that embeds some govc commands (including this one), needs to clear the array after each call.